classes/04class/exercises/c04-iam01/christina-roperto/S3readOnlyPolicy.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
```

Comment:
your answer is not wrong but if your bucket has `versioning` enabled (if you update a file, it keeps both versions as history) you won't be able to get the object as you would need the `s3:GetObjectVersion` as well, hence why the exercise requests `s3:GetObject*`

Reply:
Hi Caio, thanks for letting me know this. After a bit of checking the help list and googling, I realised that it's not so easy to update/edit policy via cli so I update via console 😬let me know if it's not ok!